.circleci/config.yml

```diff
@@ -179,7 +179,6 @@ workflows:
           <<: *run_for_all_branches
           requires:
             - build-oci-images
-            - build-all
```

Comment:
Why is it removed?

Reply:
For the E2E tests we are not testing the cli and not bulding it now it makes the tests fast.